resources/definitions/zonestar_m2_printer.def.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "ZONESTAR M2",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "ZONESTAR",
+        "manufacturer": "ZONESTAR",
+        "file_formats": "text/x-gcode",
+        "platform_offset": [ 0, 0, 0],
+        "exclude_materials": [ "zonestar_pla", "zonestar_petg", "generic_pla", "generic_petg" ],
```

Comment:
zonestar PLA is excluded? But we don't even have profiles for that. According to the info on the internet, this machine should support PLA, so is there a reason that it's excluded?

Reply:
I misunderstan the means of "exclude", we will updata it later, thanks for your check out.